html2-test-transformer.js

```diff
@@ -0,0 +1,16 @@
+const { relative, resolve } = require('path');
+
+const testRoot = resolve(__dirname, './__tests__/html2/');
+
+module.exports = {
+  process: (_, sourcePath) => {
+    const html = relative(testRoot, sourcePath);
+
+    return {
+      code: `
+        test(${JSON.stringify(html)}, () =>
+          runHTML(${JSON.stringify(`/__tests__/html2/${html}`)}));
```

Comment:
How do we do multiple uses for a single file? We leverage query parameters in some tests.

We probably could extend out transform to something like:
```html
 <!doctype html>

<test-case title="test 1 title goes here" query="?queryXXX"></test-case>
<test-case title="test 2 title goes here" query="?queryYYY"></test-case>
```
where query is optional. Then we'll need to adjust this transform to parse html and query the tags to gather available test cases.